CriteoPublisherSdk/Tests/UnitTests/Configuration/CR_DeviceInfoTests.m

```diff
@@ -112,4 +112,19 @@ - (void)testWebViewReleasedAfterUse {
   [self cr_waitForExpectations:@[ expectation ]];
 }
 
+- (void)testSafeScreenSize_GivenAnyDevice_ReturnNonZeroSizeBelowFullScreenSize {
```

Comment:
What do you think it about at some point to introduce multiple devices in tests? I think GA does that pretty well...

Reply:
Yes that would be great. With different versions too. But this may takes a really long time (or not considering that GA jobs are done in //)

Reply:
GA can do matrix testing I think... ;p